main.go

```diff
@@ -7,6 +7,8 @@ import (
 	"os"
 )
 
+const DEFAULT_TEMPLATE = "*{{build.status}}* <{{build.link}}|{{repo.owner}}/{{repo.name}}#{{truncate build.commit 8}} ({{build.branch}}) by {{build.author}}"
```

Comment:
Vet will complain about it, write DefaultTemplate instead

Reply:
Should it be DefaultTemplate or defaultTemplate?  I seem to see both but I'm not sure which convention is where.